frontend/layouts/default.tsx

```diff
@@ -1,13 +1,32 @@
+import { Header } from '@/components/shared/header'
 import { Link } from '@nextui-org/link'
+import NextHead from 'next/head'
+import Script from 'next/script'
 
 export default function DefaultLayout({
   children,
 }: {
   children: React.ReactNode
 }) {
   return (
-    <div className="relative flex flex-col h-screen">
-      <main className="container mx-auto max-w-7xl px-6 flex-grow">
+    <div className="relative flex flex-col h-screen justify-center items-center">
+      <NextHead>
+        <title>sea-bit</title>
+        <meta key="title" content={'sea-bit'} property="og:title" />
+        <meta content={'free diving booking'} property="og:description" />
+        <meta content={'free diving booking'} name="description" />
+        <meta
+          key="viewport"
+          content="viewport-fit=cover, width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"
+          name="viewport"
+        />
+        <link href="/favicon.ico" rel="icon" />
+        <link href="./output.css" rel="stylesheet" />
```

Comment:
This line generates the following warning:
`warn-once.js:16 Do not add stylesheets using next/head (see <link rel="stylesheet"> tag with href="./output.css"). Use Document instead. `

We should update its usage soon